src/utils/sha256.ts

```diff
@@ -0,0 +1,18 @@
+// from https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest
+export async function sha256(message: string | Uint8Array) {
+  // encode as UTF-8
+  const msgBuffer =
+    typeof message === "string" ? new TextEncoder().encode(message) : message;
+
+  // hash the message
+  const hashBuffer = await crypto.subtle.digest("SHA-256", msgBuffer);
```

Comment:
is this working in a node.js environment?

Reply:
Fixed in latest commit :pray: